scripts/sysman/virtualFunctionManagement.yml

```diff
@@ -0,0 +1,252 @@
+class: $sVFManagement
+name: GetVFPropertiesExp
+details:
+    - "The application may call this function from simultaneous threads."
+    - "The implementation of this function should be lock-free."
+params:
+    - type: $s_vf_handle_t
+      name: hVFhandle
+      desc: "[in] Handle for the VF component."
+    - type: $s_vf_properties_exp_t*
+      name: pProperties
+      desc: "[in,out] Will contain VF properties."
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Get memory activity stats for each available memory types associated with Virtual Function (VF)"
+class: $sVFManagement
+name: GetVFMemoryUtilizationExp
+details:
+    - "The application may call this function from simultaneous threads."
+    - "The implementation of this function should be lock-free."
+params:
+    - type: $s_vf_handle_t
+      name: hVFhandle
+      desc: "[in] Handle for the component."
+    - type: "uint32_t*"
+      name: pCount
+      desc: |
+            [in,out] Pointer to the number of VF memory stats descriptors.
+              - if count is zero, the driver shall update the value with the total number of memory stats available.
+              - if count is greater than the total number of memory stats available, the driver shall update the value with the correct number of memory stats available.
+              - The count returned is the sum of number of VF instances currently available and the PF instance.
+    - type: $s_vf_util_mem_exp_t*
+      name: pMemUtil
+      desc: |
+            [in,out][optional][range(0, *pCount)] array of memory group activity counters.
+              - if count is less than the total number of memory stats available, then driver shall only retrieve that number of stats.
+              - the implementation shall populate the vector pCount-1 number of VF memory stats.
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Get engine activity stats for each available engine group associated with Virtual Function (VF)"
+class: $sVFManagement
+name: GetVFEngineUtilizationExp
+details:
+    - "The application may call this function from simultaneous threads."
+    - "The implementation of this function should be lock-free."
+params:
+    - type: $s_vf_handle_t
+      name: hVFhandle
+      desc: "[in] Handle for the component."
+    - type: "uint32_t*"
+      name: pCount
+      desc: |
+            [in,out] Pointer to the number of VF engine stats descriptors.
+              - if count is zero, the driver shall update the value with the total number of engine stats available.
+              - if count is greater than the total number of engine stats available, the driver shall update the value with the correct number of engine stats available.
+              - The count returned is the sum of number of VF instances currently available and the PF instance.
+    - type: $s_vf_util_engine_exp_t*
```

Comment:
Similar clarification as Memory Type, here there could be multiple engineGroups per VF.
And s_vf_util_engine_exp_t seems to represent a single engine group.
Please share how to retrieve utilization of perVF engine Gropus using this API. 
(A sample would be helpful)
THanks